zipline/finance/performance/tracker.py

```diff
@@ -77,7 +78,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_calendar, env):
```

Comment:
It is no longer used for as the input of risk calculations, since the change over to using empyrical.

However, the `treasury_period_return` value is set on the `cumulative` and `period` risk metric objects.

Last I checked that value is not actually read or shown on the Quantopian platform.

However, processes which consume backtest results would have to be updated to the 'schema' change of removing that value from the risk metrics object.

We could deprecate that value by hardcoding `treasury_period_return` to 0.0. (We effectively are already doing this hardcoding on the Quantopian platform at the data level.)